Kconfig

```diff
@@ -128,7 +119,7 @@ config SECURE_BOOT
 
 config SECURE_BOOT
 	bool "Secure boot"
-	depends on IMAGE_BOOT_PG2
+	depends on IMAGE_BOOT
```

Comment:
Maybe we need to note somewhere that the security bits are not applicable to PG1 devices

Reply:
Let me check if this fits into the Kconfig help.